### ISSUE TITLE
callback not called

### ISSUE THREAD

**Issue by kolinhar** [AvianFlu/ncp#131]
When I run the first time this code with these options, it works correctly.
```JS
ncp.limit = 5;
ncp.clobber = true;

ncp("C:\\Workspaces\\myFolder", "C:\\Users\\me\\backup", err => {
    if (err) {
        return console.error(err);
    }
    console.log("Done!");
});
```

But the second time the callback is never called, I had to remove the destination folder content to make it work.

*npm -v: 5.6.0
node -v: v8.10.0*

**Comment by sundowndev** [comment]
Same issue here. Seems like the repo is not maintained anymore.

**Comment by bartlomiejzuber** [comment]
Yeah, got the same issue, I recommend to look at https://github.com/jprichardson/node-fs-extra as alternative.
